stdlib/pervasives.mli

```diff
@@ -1091,6 +1091,15 @@ val at_exit : (unit -> unit) -> unit
    The functions are called in 'last in, first out' order:
    the function most recently added with [at_exit] is called first. *)
 
+(** {6 Misc} *)
+
+val enable_runtime_warnings: bool -> unit
```

Comment:
I would rather not add things like this to pervasives. `Sys` or `Gc` may be a better place to put this. Besides it would be nice to have a getter aswell. Personally I'm not very fond of the `enable/disable` terminology. I think you could simply call these function `runtime_warnings : unit -> bool` and `set_runtime_warnings : bool -> unit`.